packages/extension-ui/src/Popup/CreateAccount/index.tsx

```diff
@@ -43,10 +58,16 @@ export default function CreateAccount (): React.ReactElement {
 
   const _onNextStep = useCallback(() => setStep((step) => step + 1), []);
   const _onPreviousStep = useCallback(() => setStep((step) => step - 1), []);
+  const _onCancel = useCallback(() => {
+    flushAccountCache().catch((e) => console.error(e));
+    onAction('/', { resetCachedAccount: true });
```

Comment:
Ummm... see what you mean. Not sure about this one at all. Will need to locally look at the logic and step through, but this seems strange.

Reply:
I got rid of this, opting for an "AccountCache" context.